sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/FullUsecasesIT.java

```diff
@@ -70,7 +70,9 @@ class ScriptCriteria {
       new ScriptCriteria("sensors-full.sqrl", "test"), //flaky (too much data)
       new ScriptCriteria("sensors-full.sqrl", "run"), //flaky (too much data)
       new ScriptCriteria("seedshop-extended.sqrl", "test"), // CustomerPromotionTest issue
-      new ScriptCriteria("seedshop-extended.sqrl", "run") // CustomerPromotionTest issue
+      new ScriptCriteria("seedshop-extended.sqrl", "run"), // CustomerPromotionTest issue
+      new ScriptCriteria("patient-sensor.sqrl", "test"), //missing data
+      new ScriptCriteria("patient-sensor.sqrl", "run") //missing data
```

Comment:
why are these excluded?

Reply:
We still benefit from them compiling and submitting the job even if they don't have any test results.